test/support/helpers/scenarios.js

```diff
@@ -140,7 +150,7 @@ module.exports.loadRemoteChangesFiles = scenario => {
 
   return glob.sync(pattern).map(f => {
     const name = path.basename(f)
-    const disabled = scenario.disabled || f.endsWith(disabledExtension)
+    const disabled = disabledScenarioTest(scenario, 'remote')
```

Comment:
Shouldn't we use `shouldSkipRemote()` here?

Reply:
We would have a dependency problem.
Both `test/support/helpers/scenario` & `test/scenarios/run` need refactoring, but this is out of the scope of this PR IMHO.